xAPI.md

```diff
@@ -509,12 +509,56 @@ __Verb__: Defines the action being done by the Actor within the Activity within
 
 ## 8.0 Getting More Out of xAPI
 
+xAPI was developed to be as lightweight as possible, but able to be focused to be powerful.  xAPI defines only 
+one verb (for voiding Statements) and only one activity type (cmi.interaction).  Obviously, this isn't meant to 
+be the extent of what xAPI can do.  xAPI is limitless in how far it can be taken, which is why this document 
+doesn't attempt to define a "starting set", because it would encourage adoption of only said set.  Many of the 
+Resources, such as State and Activity Profile, are intentionally left as "a set of name/value" pairs to not try 
+to force a certain practice.  The very notion of IRIs and suggested resolvable metadata be at the location is a 
+way to continuously increase the value of xAPI by not tying definitions down permanently.  
+
 ### 8.1 Extending xAPI
 
+xAPI can be extended in a few ways.  The most notable is Statement Extensions, which are covered in a 
+<a href="#miscext">later section</a>.  Extending Statements is how new types of information are brought into a 
+Statement and can only be done in three places.  
+
+The About Resource is another way xAPI enables extensions.  Of the Resources (located at various Endpoints), the  
```

Comment:
We should say briefly why an LRS might extend the about resources - to communicate additional information to an activity provider that wants to know about the LRS e.g. what features it might have beyond the scope of this specification.